integration/src/test/resources/org/mule/test/integration/notifications/message-processor-notification-test-flow.xml

```diff
@@ -183,7 +183,11 @@
     </flow>
 
     <flow name="idempotent-secure-hash-msg-filter">
-        <idempotent-secure-hash-message-validator idExpression="#[id]" objectStore="objectStore"/>
+        <idempotent-message-validator idExpression="%dw 2.0
+        output text/plain
+        import dw::Crypto
+        ---
+        Crypto::SHA1(variables.id)" objectStore="objectStore"/>
```

Comment:
i think the previous version was referring to the id of the event, not a variable